tinkerbell/stack/templates/nginx.yaml

```diff
@@ -183,9 +184,12 @@ spec:
   - port: {{ .Values.smee.syslog.port }}
     protocol: UDP
     name: {{ .Values.smee.syslog.name }}
+{{- end }}
+{{- if .Values.stack.relay.enabled }}
```

Comment:
port 67 for Smee is used for unicast traffic. The relay container is used for broadcast traffic. The relay being enabled should not gate this port.

Reply:
It's for when the loadbalancer IP is the same as host IP. Also this would be unnecessary when relay is disabled.
Do you have any reason why to keep it when relay is disabled?

Reply:
It points at Smee not the relay. It's for unicast DHCP messages like when a DHCP relay from a switch or router points at Smee.

Reply:
But the selector is pointing to the stack:
```yaml
selector:
    {{- with .Values.stack.selector }}
    {{- toYaml . | nindent 4 }}
    {{- end }}
```
and the stack is maintaining relay for port 67:
```yaml
- name: {{ .Values.stack.relay.name }}
  image: {{ .Values.stack.relay.image }}
  args: ["-m", "{{ .Values.stack.relay.presentGiaddrAction }}", "-c", "{{ .Values.stack.relay.maxHopCount }}", "-id", "{{ $dhcpInterfaceName }}", "-iu", "eth0", "-U", "eth0", "smee.{{ .Release.Namespace }}.svc.{{ .Values.stack.clusterDomain }}."]
  ports:
  - containerPort: 67
    protocol: UDP
    name: DHCP
```
Am I mistaken?